test/preflight/apps_v2_integration_test.go

```diff
@@ -303,6 +303,51 @@ func TestAppsV2ConfigSave_PostgresSingleNode(t *testing.T) {
     path = "/flycheck/vm"`)
 }
 
+func TestAppsV2_PostgresAutostart(t *testing.T) {
+	var (
+		err     error
+		f       = testlib.NewTestEnvFromEnv(t)
+		appName = f.CreateRandomAppName()
+	)
+
+	f.Fly("pg create --org %s --name %s --region %s --initial-cluster-size 1 --vm-size shared-cpu-1x --volume-size 1", f.OrgSlug(), appName, f.PrimaryRegion())
+
+	var machList []map[string]any
+
+	result := f.Fly("m list --json -a %s", appName)
+	err = json.Unmarshal(result.StdOut().Bytes(), &machList)
+	if err != nil {
+		f.Fatalf("failed to parse json: %v [output]: %s\n", err, result.StdOut().String())
+	}
+	require.Equal(t, 1, len(machList), "expected exactly 1 machine after launch")
+	firstMachine := machList[0]
+
+	config := firstMachine["config"].(map[string]interface{})
```

Comment:
this isn't even a nit, just a suggestion to make it easier to read :)

There's a newish API `f.MachinesList(appname)` that returns `[]api.Machine`, and that way you don't have to inline json decoding or type casting

Reply:
Thank God, I'll change it

Reply:
This should be changed in the other tests too, but that's definetely out of scope for this PR